src/types/systemIntake.ts

```diff
@@ -5,12 +5,43 @@ export type GovernanceCollaborationTeam = {
   name: string;
 };
 
-export type SystemIntakeStatus =
-  | 'DRAFT'
-  | 'SUBMITTED'
-  | 'ACCEPTED'
-  | 'APPROVED'
-  | 'CLOSED';
+export const openIntakeStatuses = [
+  'INTAKE_DRAFT',
+  'INTAKE_SUBMITTED',
+  'NEED_BIZ_CASE',
+  'BIZ_CASE_DRAFT',
+  'BIZ_CASE_DRAFT_SUBMITTED',
+  'BIZ_CASE_CHANGES_NEDED',
+  'BIZ_CASE_FINAL_NEEDED',
+  'BIZ_CASE_FINAL_SUBMITTED',
+  'READY_FOR_GRT',
+  'READY_FOR_GRB'
+];
+
+export const closedIntakeStatuses = [
+  'DRAFT',
+  'SUBMITTED',
+  'ACCEPTED',
+  'APPROVED',
+  'CLOSED'
+];
+
+export const oldIntakeStatuses = [
+  'DRAFT',
+  'SUBMITTED',
+  'ACCEPTED',
+  'APPROVED',
+  'CLOSED'
+];
+
+export const intakeStatuses = [
+  ...openIntakeStatuses,
+  ...closedIntakeStatuses,
+  ...oldIntakeStatuses
+] as const;
+
+// TODO: Remove old intake statuses once they're deprecated
+export type SystemIntakeStatus = typeof intakeStatuses[number];
```

Comment:
This is takes all the concatenated statuses and makes them a type.
This ends up looking like:
```
export type SystemIntakeStatus = 'INTAKE_DRAFT' | 'INTAKE_SUBMITTED' | 'NEED_BIZ_CASE' | ... | ... | 'APPROVED' | 'CLOSED';